Conceptpower+Spring/src/main/java/edu/asu/conceptpower/wrapper/ConceptEntryWrapper.java

```diff
@@ -17,6 +19,8 @@
  * @author Julia Damerow
  * 
  */
+@Component("conceptEntryWrapper")
+@Scope(value = "prototype")
 public class ConceptEntryWrapper implements Serializable {
```

Comment:
@jdamerow  Yes Julia, without having conceptentrywrapper as a spring bean we cannot autowire the bean in ConceptWrapperAddController search method(). If that is fine I will remove the @Component and create the object through new keyword.